### ISSUE TITLE
Inconsistent frame rate while playing, random crashes

### ISSUE THREAD

**Issue by NextLevelMemes** [mgba-emu/mgba#2223]
Hi there,

I've been trying to use this emu for Pokemon speedruns for some time now, but the frame rate doesn't seem to be consistent regardless of whether I'm recording gameplay or not. I've tried tweaking the settings but I haven't found a way to get it to work properly, whereas VBA does the job nicely. Despite having a 64 bit system the portable 32 bit build seemed to work pretty well until I tried fast-forwarding some stuff, then it started getting frame drops. 

This is the latest config I've tried in 0.9.1 that results in nice fps but keeps giving me random crashes: 
-Audio/Video: SDL, 4096 samples, 48000 Hz || OpenGL, Skip 0 frames, Native || Sync to Video, Sync to Audio, Lock aspect ratio
-Emulation: Idle loops: Run all, VBA bug compatibility
-Enhancements: Software, 1x scale

Basically I can either get frame drops or risk crashes, but not avoid both, so I'd appreciate some advice on how to get max performance to avoid frame drops if possible too.

Thanks in advance

**Comment by endrift** [comment]
Turn off sync to video. It's what's causing those crashes (it's a bug in 0.9.1).

**Comment by NextLevelMemes** [comment]
Ok, thanks, I've played a bit and it seems not to crash so far. Will try to do some run attempts later and see how it goes

Is there any other way to get it to run faster without frame skipping/fast forwarding? The runs I'm doing take around 3 hours so even slight improvements in this regard translate in a significative amount of time by the end of the run

**Comment by endrift** [comment]
Please retry on 0.9.2 which just came out.

**Comment by endrift** [comment]
Closing due to lack of reply
